orko-ui/src/selectors/coins.js

```diff
@@ -65,7 +65,14 @@ function jobTriggerMatchesCoin(job, coin) {
   )
 }
 
-function serverJobDecorate(job) {
+function orderDecorate(order, exchanges) {
+  return {
+    ...order,
+    exchangeMeta: exchanges.find(e => e.code === order.coin.exchange)
```

Comment:
I'd do this `find` once and pass it to both decorate methods.

Reply:
Yeah, but for all coins' order, still need to find for each order, so I just left it there.